### PR TITLE
Tinglis/add missed activities

### PR DESCRIPTION
**Pull request by tinglisD2L**

### CODE REVIEW

File: components/d2l-sequences-content-eol-activity-link.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+					title: {
+						type: Object,
+						computed: '_getTitle(entity)'
+					},
+					href: {
+						type: String,
+						reflectToAttribute: true,
+						notify: true
+					},
+					token: {
```
Comment (lawong-d2l) [inline components/d2l-sequences-content-eol-activity-link.html]: Could be wrong I don't think it's necessary for us to be defining token, when it is done in the EntityBehavior

File: components/d2l-sequences-content-eol-main.html [hunk 2]
```diff
@@ -82,9 +94,17 @@ <h2>
 			}
 			static get properties() {
 				return {
-					_location: {
+					href: {
+						type: String,
+						reflectToAttribute: true,
+						notify: true
+					},
+					token: {
```
Comment (lawong-d2l) [inline components/d2l-sequences-content-eol-main.html]: token defined again?

File: components/d2l-sequences-content-eol-missed.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+				return 'd2l-sequences-content-eol-missed';
+			}
+			static get properties() {
+				return {
+					href: {
+						type: String,
+						reflectToAttribute: true,
+						notify: true
+					},
+					token: {
```
Comment (lawong-d2l) [inline components/d2l-sequences-content-eol-missed.html]: token defined again?

File: components/d2l-sequences-content-eol-activity-link.html (hunk 1 above)
Comment (tinglisD2L) [reply components/d2l-sequences-content-eol-activity-link.html]: @lawong-d2l When I didn't have these before, eslint was failing. Not sure why though

**Pull request merged**